dags/bgbb.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.moz_databricks import MozDatabricksSubmitRunOperator
+from utils.mozetl import mozetl_envvar
+
+default_args = {
+    "owner": "wbeard@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 2, 22),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "wbeard@mozilla.com",
+        "amiyaguchi@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+# run on the 8th hour of the 1st day of the month
```

Comment:
I estimated this to be the time when clients_daily_v6 would be done, since this fit job is being run at a different interval as the main_summary dag.